app/main/posts/views/filters/filter-posts.html

```diff
@@ -39,7 +39,7 @@
 
     <!-- Toggle Filters show/hide -->
     <div uib-dropdown-toggle class="searchbar-options">
-        <a type="button" class="searchbar-options-filter button">
+        <a role="button" tabindex="0" type="button" class="searchbar-options-filter button">
             <sort-and-filter-counter></sort-and-filter-counter>
```

Comment:
We use the class button here, but does this element need focus? It is actually the parent-div that needs focus and toggles the dropdown so you can remove the tabindex from here.